### PR TITLE
treehouses services privatebin added

### PR DESCRIPTION
**Pull request by hiroTochigi**

### CODE REVIEW

**Comment by LordJashin32** [comment]
This errors if planet is up when running ```treehouses services privatebin up``` but this is because they are both using port 80 I assume. I took down planet and was able to get privatebin to successfully run. 

Was able to get to privatebin on port 8083 on localhost on RPi4. Seems to work. Even restarted it and it still is working. Cheers.

**Comment by hiroTochigi** [comment]
> This errors if planet is up when running `treehouses services privatebin up` but this is because they are both using port 80 I assume. I took down planet and was able to get privatebin to successfully run.
> 
> Was able to get to privatebin on port 8083 on localhost on RPi4. Seems to work. Even restarted it and it still is working. Cheers. treehouses image 116.

Did you get the following error message?
image

**Comment by kjong** [comment]
There's a mix of spaces and tabs in your code. You need to replace all the tabs to spaces. It's messing up the formatting.

File: templates/services/privatebin/privatebin_yml.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# create pihole.yml
```
Comment (kjong) [inline templates/services/privatebin/privatebin_yml.sh]: Should change to `# create privatebin.yml`

**Comment by hiroTochigi** [comment]
> Other than that I can get privatebin to run, but privatebin itself is giving me two errors:
> 
> 1. `PrivateBin requires a modern browser to work.`
> 2. `Your browser may require an HTTPS connection to support the WebCrypto API.`
> 
> Maybe these errors have something to do with the outdated docker image as you outlined in your blog post? This was something that I was thinking about when I thought about adding privatebin to services. Although that had more to do with Tor so I'm not sure if it's related or not. Regardless, we still have to think about Tor compatibility when implementing privatebin.

Only the Privatebin works via Tor Browser.
Could you try it?

**Comment by kjong** [comment]
Works perfectly on Tor.

**Pull request merged**